### PR TITLE
bug: Add back missing metrics

### PR DESCRIPTION
**Pull request by jrconlin**
A number of metrics were accidentally dropped during the code merge and refactor.

### CODE REVIEW

File: autoendpoint/src/routes/webpush.rs [hunk 1]
```diff
@@ -25,7 +26,9 @@ pub async fn webpush_route(
         RouterType::from_str(&notification.subscription.user.router_type)
             .map_err(|_| ApiErrorKind::InvalidRouterType)?,
     );
-
+    if notification.has_topic() {
+        app_state.metrics.incr("ua.notification.topic")?;
+    }
```
Comment (pjenvey) [inline autoendpoint/src/routes/webpush.rs:31]: We don't want this one, with the `ua.notification` prefix it was previously emitted by autoconnect. We consolidated it into `ua.notification.sent` in https://github.com/mozilla-services/autopush-rs/pull/330 (so I just struck it from the spreadsheet)

```suggestion
```

File: autopush-common/src/db/dynamodb/mod.rs [hunk 2]
```diff
@@ -524,6 +525,13 @@ impl DbClient for DdbClientImpl {
             )
             .await?;
 
+        let metric = self.metrics.incr_with_tags("notification.message.stored");
+        if let Some(topic) = topic {
+            metric.with_tag("topic", topic.as_str()).send();
+        } else {
+            metric.send();
+        }
+
```
Comment (pjenvey) [inline autopush-common/src/db/dynamodb/mod.rs:534]: You want to render true/false like the autoconnect code does,
```suggestion
        self.metrics.incr_with_tags("notification.message.stored")
            .with_tag("topic", message.topic.is_some().to_string()).send()
 ```

File: autopush-common/src/db/dynamodb/mod.rs [hunk 3]
```diff
@@ -543,6 +551,9 @@ impl DbClient for DdbClientImpl {
                 retryable_delete_error(self.metrics.clone()),
             )
             .await?;
+        self.metrics
+            .incr_with_tags("notification.message.removed")
```
Comment (pjenvey) [inline autopush-common/src/db/dynamodb/mod.rs:555]: We call it `deleted` in the other impl, unless you want to rename it:
```suggestion
            .incr_with_tags("notification.message.deleted")
```

**Review by pjenvey — CHANGES_REQUESTED** [review]

File: autopush-common/src/db/dynamodb/mod.rs (hunk 3 above)
Comment (jrconlin) [reply autopush-common/src/db/dynamodb/mod.rs]: updated the spreadsheet to include `notification.message.deleted` 
thanks!

File: autopush-common/src/db/dynamodb/mod.rs [hunk 4]
```diff
@@ -511,6 +511,7 @@ impl DbClient for DdbClientImpl {
     }
 
     async fn save_message(&self, uaid: &Uuid, message: Notification) -> DbResult<()> {
+        let topic = message.topic.clone().is_some().to_string();
```
Comment (pjenvey) [inline autopush-common/src/db/dynamodb/mod.rs:514]: nit:
```suggestion
        let topic = message.topic.is_some().to_string();
```

**Review by pjenvey — CHANGES_REQUESTED** [review]

**Review by pjenvey — APPROVED** [review]

**Pull request merged**